index.html

```diff
@@ -4,10 +4,33 @@
 		<meta charset="UTF-8" />
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<title>Document</title>
+		<title>KC Portfolio</title>
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 		<link rel="stylesheet" href="styles/style.css" />
 	</head>
 	<body>
-		<h1>Hello Microverse!</h1>
+		<header class="mobile-head">
+            <div class="mobile-version-head-accent">
+                <div class="head-img"><img src="images/image_geometry_1.svg" alt="geometric accent design "></div>
+                <div class="hamBtn">
+                    <div class="ham1 ham"></div>
+                    <div class="ham2 ham"></div>
+                    <div class="ham3 ham"></div>
+                </div>
```

Comment:
- [ ] For semantic reasons, kindly make sure your hamburger menu is wrapped inside a `nav` tag and not a generic `div` since it's a navigation bar.